src/components/Modal/FeedbackModal/index.tsx

```diff
@@ -0,0 +1,156 @@
+import Modal from "..";
+import * as S from "./styled";
+import Close from "@assets/icons/close.svg";
+import SosoOn from "@assets/icons/soso-on.svg";
+import SosoOff from "@assets/icons/soso-off.svg";
+import BadOn from "@assets/icons/bad-on.svg";
+import BadOff from "@assets/icons/bad-off.svg";
+import GoodOn from "@assets/icons/good-on.svg";
+import GoodOff from "@assets/icons/good-off.svg";
+import { useState } from "react";
+import { instance } from "../../../apis/axios";
+
+interface FeedbackModalProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+type ToggleState = {
+  good: boolean;
+  soso: boolean;
+  bad: boolean;
+};
+
+type IconType = keyof ToggleState;
+
+type FeedbackType = "BAD" | "AVERAGE" | "EXCELLENT";
+
+interface FeedbackRequest {
+  type: FeedbackType;
+  message: string;
+}
+
+const MAX_LENGTH = 1000;
+
+const FeedbackModal: React.FC<FeedbackModalProps> = ({
+  isOpen,
+  closeModal,
+}) => {
+  const [toggleStates, setToggleStates] = useState<ToggleState>({
+    good: false,
+    soso: false,
+    bad: false,
+  });
+
+  const handleToggle = (iconType: IconType): void => {
+    setToggleStates((prev) => {
+      // 이미 선택된 아이콘을 다시 클릭한 경우
+      if (prev[iconType]) {
+        return {
+          ...prev,
+          [iconType]: false,
+        };
+      }
+
+      // 다른 아이콘을 클릭한 경우
+      return {
+        good: false,
+        soso: false,
+        bad: false,
+        [iconType]: true, // 새로 선택된 아이콘만 on
+      };
+    });
+  };
+
+  const [value, setValue] = useState("");
+
+  const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const newValue = e.target.value;
+    if (newValue.length <= MAX_LENGTH) {
+      setValue(newValue);
+    }
+  };
+
+  const getFeedbackType = (): FeedbackType => {
+    if (toggleStates.good) return "EXCELLENT";
+    if (toggleStates.soso) return "AVERAGE";
+    if (toggleStates.bad) return "BAD";
+    return "AVERAGE";
+  };
```

Comment:
```suggestion
  const getFeedbackType = (feedbackState: ToggleState): FeedbackType => {
    if (feedbackState.good) return "EXCELLENT";
    if (feedbackState.soso) return "AVERAGE";
    if (feedbackState.bad) return "BAD";
    return "AVERAGE";
  };
```
순수 함수 측면에서 전역 상태 값을 사용하기 보다는 인자로 state를 받아서 같은 인자가 들어오면 항상 같은 값을 리턴하여 순수하게 동작하는게 더 좋을 것 같아요~